### ISSUE TITLE
Respect `RUST_LOG` environment variable

### ISSUE THREAD

**Issue by orhun** [gin66/tui-logger#35]
Is this possible to change the default logging level based on the value of `RUST_LOG` environment variable? I tried setting it but it had no effect.

**Comment by gin66** [comment]
`RUST_LOG` is not supported by tui_logger and IMHO only used by [env_logger](https://github.com/rust-cli/env_logger).

Nevertheless, the application could extract the info from RUST_LOG and initialize tui_logger with several calls of `tui_logger::set_level_for_target()`. If there is a good PR with opt-in possibility (e.g. feature), then tui_logger can be extended accordingly.

**Comment by orhun** [comment]
Got it. I did the following for now:

```rs
    tui_logger::init_logger(if let Ok(log_level) = std::env::var("RUST_LOG") {
        LevelFilter::from_str(&log_level)?
    } else {
        LevelFilter::Trace
    })?;
```

You are right about the `env_logger` part. I started to implement this feature for `tui-logger` and after some point I realized I was writing identical code to `env_logger`. So it seems like it is best to keep this feature separate for now.

BTW are you thinking of releasing a new version soon? I'm waiting for #33 and #34 to use them in my application 🐻
